### ISSUE TITLE
Train/val loss on flickr30k

### ISSUE THREAD

**Issue by yjsong22** [jmisilo/clip-gpt-captioning#77]
Hi,
Thanks for your nice work. I want to ask what are the training and validation losses for flickr 30k dataset in your experiments. I experimented with your code on flickr 30k but it seems to be overfitting and the losses are very high (about 10.7). Do you have any idea on this?
Thank you.

**Comment by jmisilo** [comment]
Hey, so the train loss was 10.03 and the validation loss was 10.02. These are quite high, but the challenging part was reducing them + computing power was limited. I don't have any good ideas now how to improve this, maybe playing with `forward_expansion` and `num_layers` layers of `Mapping`?
